### DISCUSSION TITLE
Questions regarding test errors after installation

### DISCUSSION THREAD

**Discussion by jsyoonsw** [idaholab/moose#26937] [Q&A Getting Started]
Hello,

I am a new user of moose, and currently trying to compile and install the software on my local MAC OS. By the way, after the installation and testing following the instructions on the manual, I encoutered 7 failure errors. I am not sure which can be ignored among them. Would you give me any advice on these issues? I really appreciate your help in advance!

<img width="779" alt="1" src="https://github.com/idaholab/moose/assets/161783804/3e1fa689-f7d7-4ecb-bca8-f77d2088d315">
<img width="1198" alt="Screenshot 2024-02-29 at 11 31 54 PM" src="https://github.com/idaholab/moose/assets/161783804/0786fc1a-1cf9-4b1d-8ea0-50f2a62264d3">
<img width="1196" alt="Screenshot 2024-02-29 at 11 32 39 PM" src="https://github.com/idaholab/moose/assets/161783804/d360b7c1-1d63-47e7-9
<img width="1199" alt="Screenshot 2024-02-29 at 11 33 18 PM" src="https://github.com/idaholab/moose/assets/161783804/1aa289ce-35fd-46b4-8cf1-152515182c3e">
981-31a03a699280">
<img width="1195" alt="Screenshot 2024-02-29 at 11 31 14 PM" src="https://github.com/idaholab/moose/assets/161783804/da2bcfb0-ece4-49e2-9595-b02c85d7a364">
<img width="1198" alt="Screenshot 2024-02-29 at 11 31 35 PM" src="https://github.com/idaholab/moose/assets/161783804/6eb7c147-4953-4192-9c78-9012fada76df">
<img width="1036" alt="Screenshot 2024-02-29 at 11 33 50 PM" src="https://github.com/idaholab/moose/assets/161783804/821bf6cd-28f9-495f-8696-c27fed01e450">

**Comment by GiudGiud** [comment]
Hello

These look related to the parser. It s not clear why.
I would ignore them since the parser worked fine for the 4000 other tests

Guillaume

**Reply by jsyoonsw** [reply]
Hello Guillaume,

Thanks for your reply.
As you suggested, I ignored the error message and moved on to the creating and building application.
But, I encoutered another error messages while building my app "lmssb", indicating that there are some missing files and directories. Do you think it has something to do with the parser error? or should I check other possibilities?

I appreciate your help.

Best,
JS

<img width="772" alt="Screenshot 2024-03-04 at 11 58 00 PM" src="https://github.com/idaholab/moose/assets/161783804/dc7a8d81-eb5c-4a63-82ce-df833a4f6ba2">

**Reply by GiudGiud** [reply]
You just need to `conda activate moose` every time

**Reply by jsyoonsw** [reply]
Hello Guillaume,

Thanks for your reply.
After building an app in the moose environment, it works!
But, can I ask additional questions in this thread?

The main reason I installed the moose is to use the EEL Moose tool developed in Argonne national lab (https://github.com/hugary1995/eel). When I followed all instructions to build an application within Moose and test it, I encountered the errors like below. Could you give me any advice for this?
(I am not sure whether these questions need to be directed to the EEL developer or here... Thanks for your understanding)

Thank you very much!

Best,
JS

<img width="779" alt="Screenshot 2024-03-06 at 12 22 04 AM" src="https://github.com/idaholab/moose/assets/161783804/973e3a8b-3a3c-48c0-8b29-a6d3fd09501e">
<img width="2496" alt="Screenshot 2024-03-06 at 12 25 19 AM" src="https://github.com/idaholab/moose/assets/161783804/9faee1e6-65c5-4c04-bdff-0de15232c61d">
<img width="2516" alt="Screenshot 2024-03-06 at 12 26 03 AM" src="https://github.com/idaholab/moose/assets/161783804/5ba0767d-fd6e-45db-8503-d2dd891edaa1">
<img width="2470" alt="Screenshot 2024-03-06 at 12 27 18 AM" src="https://github.com/idaholab/moose/assets/161783804/6a736e52-32d6-44e6-a80d-67fd42f83cae">
<img width="2390" alt="Screenshot 2024-03-06 at 12 27 39 AM" src="https://github.com/idaholab/moose/assets/161783804/f1b02788-4d0f-48bc-bc2e-72b85bcf93ac">

**Reply by GiudGiud** [reply]
@hugary1995 
If you want to fix this, compile in debug mode with `METHOD=dbg make -j6`
then you should get a more helpful error for these tests

**Reply by hugary1995** [reply]
@jsyoonsw Thanks for your interest in EEL. Most likely the crashes are due to outdated test input files. I should be able to find some time to fix them. I am currently occupied with some other work, and I should be able to get to this by the end of next week. If not, feel free to send me another reminder.

**Reply by jsyoonsw** [reply]
Hello Guillaume, 
Thanks for your suggestion and connecting me to hugary1995!

Hello hugary1995,
I appreciate your explanation regarding the crashes that I encountered during the EEL test.
Then, I will wait for your update on EEL.
Thank you.
